tests/unit/ligo-plugin-tests/ligo-plugin-tests.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ___TEST___ } from '../../../taqueria-plugin-ligo/compile';
```

Comment:
Why don't we put LIGO tests in in the LIGO plugin?

Reply:
harder to configure, harder to find.